src/AbstractResourceListener.php

```diff
@@ -252,7 +253,7 @@ public function fetch($id)
     /**
      * Fetch all or a subset of resources
      *
-     * @param  array $params
+     * @param  array|Parameters $params
```

Comment:
So far, so good: question is whether we have a test scenario checking this use-case?

Reply:
Hang on, I'll check what's in the test folder ;-)